addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/profiles/EnOceanProfiles.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.enocean.profiles;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.profiles.ProfileTypeUID;
+import org.eclipse.smarthome.core.thing.profiles.TriggerProfileType;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+
+/**
+ * The {@link EnOceanProfiles} hold the profiles types and profile type UIDs of all EnOcean profiles.
+ *
+ * @author Jan Kemmler - Initial contribution
+ */
+@NonNullByDefault
+public interface EnOceanProfiles {
+
+    ProfileTypeUID ROCKER_TO_ON_OFF = new ProfileTypeUID("enocean", "rocker-to-ONOFF");
```

Comment:
maybe you want to extract the string "enocean" and "rocker-to-ONOFF" into constants. Same goes for the other cases where you construct UIDs.